locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
 	"Blockquote": "Blockquote",
 	"Bold font": "Bold font",
 	"Italics font": "Italics font",
 	"Strikethrough": "Strikethrough",
 	"Inserted text": "Inserted text",
-	"Marked text": "Marked text",
+	"Marked text": "Highlighted text",
 	"Link": "Link",
 	"Image": "Image",
 	"Code": "Code",
 	"Externals": "Externals",
-	"This is a alert area.": "This is a alert area.",
-	"Revert": "Revert",
+	"This is a alert area.": "This is an alert area.",
+	"Revert": "Undo",
```

Comment:
This is not an undo feature.
You will revert the note to specific revision.
Undo should specifically indicate the editor "undo" action.
I would like to avoid using the same keyword on different features.